docs/example-react-redux.md

```diff
@@ -23,7 +23,7 @@ class Counter extends React.Component {
         <h2>Counter</h2>
         <div>
           <button onClick={this.decrement}>-</button>
-          <span data-testid="count-value">{this.props.count}</span>
+          <h1>{this.props.count}</h1>
```

Comment:
h1 after h2? Let's leave this alone.

Reply:
My bad, missed it sorry.. Will fix.

Reply:
@alexkrolick Fixed it, thanks..
Also refactored the class component to a functional one in the current time spirit :)